app/src/views/Actors.vue

```diff
@@ -289,6 +356,17 @@
         </v-card-actions>
       </v-card>
     </v-dialog>
+
+    <v-dialog v-model="deleteSelectedActorsDialog" max-width="400px">
+      <v-card>
```

Comment:
Create a `deleteSelectionLoader` bool, activate it when running the delete mutation (and use it in other areas, see other comments)

```suggestion
      <v-card :loading="deleteSelectionLoader">
```

Reply:
In practice, it might not be necessary if the deletion is so fast that we wouldn't even see the loader. I don't know how fast actor deletion is though